spl/token/_layouts.py

```diff
@@ -0,0 +1,73 @@
+"""Token instruction layouts."""
+from enum import IntEnum
+
+from construct import Int8ul, Int64ul, Pass  # type: ignore
+from construct import Struct as cStruct
+from construct import Switch
+
+from solana._layouts.shared import PUBLIC_KEY_LAYOUT
+
+
+class InstructionType(IntEnum):
+    """Token instruction types."""
+
+    InitializeMint = 0
+    InitializeAccount = 1
+    InitializeMultisig = 2
+    Transfer = 3
+    Approve = 4
+    Revoke = 5
+    SetAuthority = 6
+    MintTo = 7
+    Burn = 8
+    CloseAccount = 9
+    FreezeAccount = 10
+    ThawAccount = 11
+    Transfer2 = 12
+    Approve2 = 13
+    MintTo2 = 14
+    Burn2 = 15
+
+
+_INITIALIZE_MINT_LAYOUT = cStruct(
+    "decimals" / Int8ul,
+    "mint_authority" / PUBLIC_KEY_LAYOUT,
+    "freeze_authority_option" / Int8ul,
+    "freeze_authority" / PUBLIC_KEY_LAYOUT,
+)
+
+_INITIALIZE_MULTISIG_LAYOUT = cStruct("m" / Int8ul)
```

Comment:
can we have a better name for `m` ?

Reply:
Wanted to be consistent with https://docs.rs/spl-token/2.0.3/spl_token/instruction/enum.TokenInstruction.html and the js lib.